CMakeLists.txt

```diff
@@ -204,11 +203,6 @@ if (NO_EXCEPTIONS)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-exceptions")
 endif()
 
-if (USE_CLANG_SANITIZERS)
```

Comment:
This is not a formatting fix, but ok.

Reply:
Yeah, you are right. I'll extract it to a separate commit.